openssl/README.md

```diff
@@ -13,3 +13,82 @@ Binary package
 ## Usage
 
 *TODO: Add instructions for usage*
+
+## Stage 1 Plan
+
+This section outlines a general playbook for maintainers who are building this plan.
+
+As a stage 1 plan, this plan has extra considerations when merging PRs and building the plan. This stage 1 plan can safely be built without a core plans base refresh.
+
+1. The following plans must be verified as successfully building, along with success of any automated tests, before merging any PRs:
```

Comment:
In line with @predominant's  comment,  think it's important to call out here _why_ we need to build software from the habitat and builder repos as part of validation of changes.   In short,  because Habitat is a self-hosting system (we use hab to build hab and core-plans, and core-plans to build hab) we need to perform an extra level of verification so that a change in the base-plans doesn't prevent our ability to ship.